packages/eslint-plugin-i18n/src/rules/no-unknown-key.js

```diff
@@ -1,6 +1,52 @@
+const _ = require('lodash');
 const minimatch = require('minimatch');
 const { getKeyValue, get, has, getLangConfig, areWeUsingUseTranslate } = require('../utils/utils');
 
+const check = (key, countNode, config, langsKey, context, node) => {
```

Comment:
`config` is in third here

Reply:
Can we keep consistency ? or Is it a linter convention ?

Reply:
Or we can use an object with named keys ... 6 params function starts to have high probability of mistakes 😅

Reply:
✅